src/components/Modal/node/SendMessageModal.tsx

```diff
@@ -67,6 +75,28 @@ type SendMessageModalProps = {
   tooltip?: JSX.Element | string;
 };
 
+function sortAddresses(
```

Comment:
would be good to document the order you are trying to achieve here.

Reply:
good idea